mealz-shared-analytics/scripts/generate_ts_props_interface.rb

```diff
@@ -0,0 +1,48 @@
+require 'fileutils'
+
+# Méthode pour convertir une chaîne en PascalCase
+def camelize(str)
```

Comment:
un fonction qui convertir une chaîne en PascalCase qui s'appelle camelize ?

Reply:
mdr oui bon

Reply:
je suppose que cest à changer cette fonctione & c'est bon

Reply:
fixed in next pr